solr/server/solr/configsets/sample_techproducts_configs/conf/solrconfig.xml

```diff
@@ -674,12 +679,12 @@
          *** WARNING ***
          Before enabling remote streaming, you should make sure your
          system has authentication enabled.
-
-    <requestParsers enableRemoteStreaming="false"
+       -->
+    <requestParsers enableRemoteStreaming="true"
```

Comment:
Why?

Reply:
This was required for one of the demos..   Line 113 of script-update-processor shows a curl command:

```
http://localhost:8983/solr/techproducts/update?commit=true&stream.contentType=text/csv&fieldnames=id,description&stream.body=1,foo&update.chain=script
```

If this is too dangerous, I could revert this change and document the need to make the change in the directions.  It's just one more barrier to easily trying the feature with the tech products example.

Reply:
Ok.... "sample_techproducts_configs" being something that would be just for demo/experimentation locally and never a shipping config.  I don't see why anyone would foolishly copy this particular configSet (in entirety) as the basis for their own either.